### DISCUSSION TITLE
Meteor Client Bundler deprecated?

### DISCUSSION THREAD

**Discussion by JacquesRobin** [quasarframework/quasar#13813] [General - Components / Directives / etc]
Following tips from [ontwikkelfabriek](https://github.com/ontwikkelfabriek)
[dobbel](https://github.com/ontwikkelfabriek)  among others I am trying to build an app with Quasar+Vue in the front-end and Meteor in the back-end using the Quasar CLI to build the front-end and Urigo's Meteor-Client-Bundler to build the back-end.

When I do: 
npm install -g meteor-client-bundler, 
I get the following warning:

> npm WARN deprecated core-js@2.6.12: core-js@<3.23.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Some versions have web compatibility issues. Please, upgrade your dependencies to the actual version of core-js.

Does this mean that the meteor-client-bundler is no longer maintained and no longer usable with the current Meteor version 2.7.4?

**Comment by TobyMosque** [comment]
https://github.com/Urigo/meteor-client-bundler/issues

**Reply by JacquesRobin** [reply]
Are you suggesting that the answer to my questions is obviously yes due to the fact that the last time an issue got a response on the meteor-client-bundler repository was 3 years ago?

**Reply by TobyMosque** [reply]
no, I'm suggesting you would ask to the Meteor-Client-Bundler maintainer, but i think u already got your answer.
